test/dynamo/test_view.py

```diff
@@ -0,0 +1,42 @@
+# Owner(s): ["module: dynamo"]
+import torch
+
+import torch._dynamo
+import torch._dynamo.test_case
+
+torch._dynamo.config.capture_scalar_outputs = True
```

Comment:
Don't do this, this will interfere with other tests if they're running same process (as is the case if you do pytest.) Instead, use the decorator `@torch._dynamo.config.patch` to temporarily change config. Or if you put these tests in test_unspec.py, the class globally has a patch for all tests in that file.

Reply:
Fixed. Please take a look.